esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -156,8 +156,23 @@ diagnostics:
     scripts: null
 
 
-  ESACCI-AEROSOL:
-    description: ESACCI-AEROSOL check
+#  ESACCI-AEROSOL:
```

Comment:
I think also here it would be nice to have an additional comment that describes why the collection of lines were commented out.